src/FSharp.Data.GraphQL.Server.Middlewares/AssemblyInfo.fs

```diff
@@ -0,0 +1,17 @@
+﻿// Auto-Generated by FAKE; do not edit
+namespace System
+open System.Reflection
+
+[<assembly: AssemblyTitleAttribute("FSharp.Data.GraphQL.Server.Middlewares")>]
```

Comment:
Not sure about this file. Should we have an assembly info in each project even though it is generated by the build?